app/services/invite_schools.rb

```diff
@@ -230,6 +230,34 @@ def send_year2020_invite_email
     end
   end
 
+  def feature_flag_and_send_participant_validation_beta_emails(array_of_urns:)
+    start_url = participant_validation_start_url
+    user_research_url = participant_validation_research_url
+
+    School.where(urn: array_of_urns).find_each do |school|
+      FeatureFlag.activate(:participant_validation, for: school)
+
+      school.active_ecf_participant_profiles.each do |profile|
+        if profile.ect?
+          SchoolMailer.participant_validation_ect_email(
+            recipient: profile.user.email,
+            school_name: school.name,
+            start_url: start_url,
+            user_research_url: user_research_url,
+          ).deliver_later
+        elsif profile.mentor?
+          SchoolMailer.participant_validation_fip_mentor_email(
+            recipient: profile.user.email,
+            school_name: school.name,
+            start_url: start_url,
+          ).deliver_later
+        end
```

Comment:
Should we check that the school is actually doing FIP before we send this emails? CIP schools have mentors too, couple hundreds of them.

Reply:
Or should we perhaps make the email work with both cip and fip mentors? Probably a product question which one we want (or maybe we want something else).

Reply:
Not sure how much of an issue it is at this point as this is to email a pre-selected set of FIP schools for the beta, but generally we should do this.

Reply:
Added a check for this just to be on the safe side.